tests/lib/snaps/test-snapd-service/bin/start

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+echo $(snapctl get configvalue) > $SNAP_DATA/configvalue
```

Comment:
We don't need the _echo_ if we also drop the _$(...)._

The use of "configvalue" is also a bit of a strange name for a configuration key (key, not value). Can we use something like "service-option" instead, hinting at the fact this is targeting service use?